__tests__/contacts/contacts.mutation.test.js

```diff
@@ -0,0 +1,380 @@
+/* eslint-disable object-curly-newline */
+/* eslint-disable no-undef */
+const { gql } = require('apollo-boost');
+const client = require('../../utils/apollo-test-client');
+const {
+  CONTACT_ALREADY_EXIST,
+  CONTACT_NOT_FOUND,
+} = require('../../error-messages/contacts.messages');
+
+const contact = {
+  phoneNumber: '1241241242144',
+  openHours: [
+    { lang: 'uk', value: 'ПН ...' },
+    { lang: 'en', value: 'FR ...' },
+  ],
+  address: [
+    { lang: 'uk', value: 'Вулиця 3' },
+    { lang: 'en', value: 'Street 3' },
+  ],
+  email: 'test@test.com',
+  images: {
+    medium: 'medium.jpg',
+  },
+  link: 'https://testURL.com',
+};
+
+const updatedContact = {
```

Comment:
Please, move this contact variable to the `contact.variables.js` and use it in query tests and mutation tests